### DISCUSSION TITLE
Thank you for reviving this wonderful piece of software

### DISCUSSION THREAD

**Discussion by solluh** [AlfHou/hammond#17] [General]
I hope that the community's suggestions will be considered in this fork. Too bad that akhilrex abandoned the project. The German translation is a great start, thanks for that, boerniee!

The data transfer was problem-free, as always with Docker.

Solluh

**Comment by AlfHou** [comment]
That's great to hear @solluh! If you have the free time, it would be great if you added a section to the Readme about what you did to migrate :)

**Reply by solluh** [reply]
> what you did to migrate

Migration is such a big word for backing up the akhilrex-hammond folder and pulling your new fork...

**Reply by arnovdk** [reply]
Another +1 for a trouble free migration on my end. And as solluh mentioned; it just boils down to backing up the volume and using the new image.

I'm excited to see that the project has gotten a bit of fresh air and where things will develop from here.
